### PR TITLE
SPUI-1441 > Fix Help Center icon + header

### PR DESCRIPTION
**Pull request by VovaMedvediev**

### CODE REVIEW

File: lib/app/screens/account/components/help_center_web_view.dart [hunk 1]
```diff
@@ -4,21 +4,21 @@ import 'package:simple_kit/simple_kit.dart';
...
     required this.link,
   }) : super(key: key);
 
   final String link;
 
   static void push(
-      BuildContext context,
-      String link,
-      ) {
+    BuildContext context,
```
Comment (eli1stark) [inline lib/app/screens/account/components/help_center_web_view.dart:16]: use named parameters here, it's more readable

File: lib/app/screens/account/account.dart [hunk 2]
```diff
@@ -148,10 +148,13 @@ class Account extends HookWidget {
                           ),
                           SimpleAccountCategoryButton(
                             title: 'Help Center',
-                            icon: const SFaqIcon(),
+                            icon: const SQuestionIcon(),
                             isSDivider: true,
                             onTap: () {
-                              navigatorPush(context, FaqWebView(link: faqLink));
+                              navigatorPush(
```
Comment (eli1stark) [inline lib/app/screens/account/account.dart:154]: use HelpCenterWebView.push(a1: a1, a2: a2) instead

**Review by eli1stark — CHANGES_REQUESTED** [review]

File: lib/app/screens/account/components/help_center_web_view.dart (hunk 1 above)
Comment (VovaMedvediev) [reply lib/app/screens/account/components/help_center_web_view.dart:16]: refactored

File: lib/app/screens/account/account.dart (hunk 2 above)
Comment (VovaMedvediev) [reply lib/app/screens/account/account.dart:154]: refactored

**Pull request merged**